FreeRTOS/FreeRTOS
# Add register test tasks to QEMU project

Description
-----------
Updates the CORTEX_MPS2_QEMU_IAR_GCC GCC and IAR builds to include the register test tasks.

Test Steps
-----------
Clone a clean copy of the repo, then make sure the CORTEX_MPS2_QEMU_IAR_GCC/build/IAR embedded workspace and CORTEX_MPS2_QEMU_IAR_GCC/build/gcc Eclipse workspace project build.




By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
